optuna/study.py

```diff
@@ -751,8 +751,12 @@ def _run_trial(
 
         return trial
 
-    def _log_completed_trial(self, trial, result):
-        # type: (trial_module.Trial, float) -> None
+    def _log_completed_trial(self, trial: trial_module.Trial, result: float) -> None:
+
+        if logging.get_verbosity() > logging.INFO:
```

Comment:
We could also use https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor. It internally uses a cache (https://github.com/python/cpython/blob/26a1ad1c24717990265b71ed093d691500d6301c/Lib/logging/__init__.py#L1695).
```suggestion
        if not _logger.isEnabledFor(logging.INFO):
```